src/utils/sheet-check.ts

```diff
@@ -22,19 +22,24 @@ export async function CheckUserKeyStatus(interaction: CommandInteraction) {
     }
     await interaction.deferReply({ ephemeral: true });
 
+    const hasCustomUser = interaction.options.getUser('user') != null;
```

Comment:
specify that the user option is required and it will never be null

Reply:
This is null so we return a bool. Keep in mind this function is used both in the normal check command and in the checkuser command, so this allows us to have a bool to check whether we've passed in a custom user or not

Reply:
rip